alternator/serialization.cc

```diff
@@ -59,7 +59,9 @@ struct from_json_visitor {
         bo.write(t.from_string(rjson::to_string_view(v)));
     }
     void operator()(const bytes_type_impl& t) const {
-        bo.write(rjson::base64_decode(v));
+        // FIXME: it's difficult at this point to get information if value was provided
```

Comment:
If I understand correctly, this from_json_visitor code gets called only during **write** operations, so I think it's fine that a write operation fails with an exception if something is wrong in what it tries to write (regardless of where this content comes from). So I think this "true" is correct.

Reply:
Yes I also think it is. I'd leave the fixme as the code can change and suddenly it won't be correct, `from_json_visitor` seems to be a generic code.